src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 500;
+    this.name = 'Test';
+    Animal.alive.push(this.name);
+  }
+  static deleteIndex = Animal.alive.findIndex((animal) => animal.health !== 'dead');
```

Comment:
The static deleteIndex property is not used and is not part of the task requirements. Consider removing it if it's not needed.